pylint/extensions/_check_docs_utils.py

```diff
@@ -159,6 +162,111 @@ def possible_exc_types(node: nodes.NodeNG) -> set[nodes.ClassDef]:
         return set()
 
 
+def _is_ellipsis(node: astroid.NodeNG) -> bool:
+    if not PY38_PLUS:
+        return isinstance(node, astroid.Ellipsis)
+
+    return isinstance(node, astroid.Const) and node.value == Ellipsis
+
+
+def _merge_annotations(
+    annotations: Iterable[astroid.NodeNG], comment_annotations: Iterable[astroid.NodeNG]
```

Comment:
We should probably follow the style of importing from `nodes` instead of `astroid` here.